packages/consent/consent-wrapper-onetrust/src/domain/wrapper.ts

```diff
@@ -1,25 +1,30 @@
 import {
-  AnyAnalytics,
-  Categories,
   createWrapper,
   CreateWrapperSettings,
   resolveWhen,
 } from '@segment/analytics-consent-tools'
 
 import {
+  getNormalizedCategoriesFromGroupData,
+  getNormalizedCategoriesFromGroupIds,
   getConsentedGroupIds,
-  getGroupData,
   getOneTrustGlobal,
 } from '../lib/onetrust-api'
 
-interface OneTrustOptions {
+export interface OneTrustSettings {
   integrationCategoryMappings?: CreateWrapperSettings['integrationCategoryMappings']
+  disableConsentChangedEvent?: boolean
 }
 
+/**
+ *
+ * @param analyticsInstance - An analytics instance. Either `window.analytics`, or the instance returned by `new AnalyticsBrowser()` or `AnalyticsBrowser.load({...})`
+ * @param settings - Optional settings for configuring your OneTrust wrapper
+ */
 export const oneTrust = (
```

Comment:
nit: should this be named `withOneTrust` instead? To make sure people understand it's wrapping the `analytics` object?

Reply:
I've been weighing that change as well, I'm glad you chimed in